frontend/src/components/landing/Citation.vue

```diff
@@ -47,8 +57,14 @@
 
 <template>
   <div class="summary-section" id="citation">
-    <div v-if="isVolatile" class="volatile-disclaimer-banner">
-      Be aware that this data is volatile and may be updated in the future.
+    <div
+      v-if="isVolatile || isLegacy"
+      class="disclaimer-banner"
+      :class="isVolatile ? 'volatile-banner' : 'legacy-banner'"
+    >
+      <img class="banner-icon" alt="warning icon" :src="require('../../assets/icons/test-warning.svg')" />
+      <span v-if="isVolatile">This data set is volatile and may be updated in the future.</span>
+      <span v-if="isLegacy">This data set is produced using nonstandard processing.</span>
```

Comment:
Olisiko "was produced" tässä parempi? Pitäisikö myös puhua "file":istä, koska eikö "data set" tarkoittanut vuoden mittauksia ACTRIS:ssa?

Reply:
Joo `was produced` on oikea aikamuoto tässä. Vaihtoehdot on varmaankin:

- This file was..
- These data were..
- This data object was..
- This data set was.. (ehkä ei tosiaan kannata käyttää tätä)

En tiedä mikä olisi paras. :confused: